client/payment-details/summary/index.tsx

```diff
@@ -285,7 +285,7 @@ const PaymentDetailsSummary: React.FC< PaymentDetailsSummaryProps > = ( {
 							</p>
 							<div className="payment-details-summary__breakdown">
 								{ renderStorePrice ? (
-									<p>
+									<p className="payment-details-summary__breakdown__explicit-currency">
```

Comment:
Maybe `__settlement-currency` here instead of `__explicit-currency`, since explicit currency is a way of currency formatting that could be used for any currency field and this field displays "Charge amount in settlement currency" ?

Reply:
Good suggestion, made change here: https://github.com/Automattic/woocommerce-payments/pull/8273/commits/4769665e6ba4447ae1a4c67ffec2726591655b09